releasenotes/notes/add-elide-swaps-b0a4c373c9af1efd.yaml

```diff
@@ -0,0 +1,35 @@
+---
+features:
+  - |
+    Added a new optimization transpiler pass, :class:`~.ElideSwaps`,
+    which is designed to run prior to the :ref:`layout_stage` and will
+    optimize away any :class:`~.SwapGate`\s in a circuit by permuting virtual
+    qubits. For example, taking a circuit with :class:`~.SwapGate`\s:
```

Comment:
Shall we mention that routing will still add required swaps into the circuit but that this strictly enables more opportunity for improvements in routing?  That might be too much detail for the note, though.